webpack/scenes/Content/ContentConfig.js

```diff
@@ -0,0 +1,11 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export default () => [
+  {
+    names: { title: __('Python Packages'), plural: 'python_packages', singular: 'python_package' },
```

Comment:
is there any way we can have this information pulled from the api?

Reply:
Just the names or all of the information? Having the separate config file for this might be necessary to allow different content types to render information with different react components.

Reply:
just the names, but potentially the columns in the future as well.  @sjha4 do you have any ideas how we could do this?  the idea is still that we can drop in the repository_type definition on the ruby server and thats it, the UI will adapt as needed

Reply:
I don't know how much variation in the react components is expected, but maybe when a react component is desired it can just be associated with a keyword from the api.

Reply:
I like this approach of having a UI config file to control what shows on the UI..

But if we want to make it more generic, we could do a default list of columns to display (name, version) to begin with and get this config file to loop over content types and form this config automatically. We could still override a content config if we wanted to.

Reply:
I'm not necessarily opposed to it, it just increases the footprint of where things need to be added.  If we do go this route, we'll need something that disables content types that aren't actually enabled on the server (for example, we wouldn't want to show python packages if python isn't enabled)

Reply:
What if I add a kind of default that can have column headers added to it from the registration file, this way the only time it would need to be reconfigured on the javascript side is if different react components need to be specified.

And I'll make the default selected type whatever the first type returned from the API is, rather than python packages specifically.

Reply:
> And I'll make the default selected type whatever the first type returned from the API is, rather than python packages specifically.

This will probably need to be plucked out of the URL in the future..
For example: If user is going to ostree nav link, url changes to /ostree and that'd be the selection in the dropdown..

Reply:
In that case, when someone clicks "Other Content Types" in the navbar, should it go to `/content/:content_type` or `/:content_type`, rather than just `/content`?

Reply:
/content is fine for now..We will need to do some changes when we route explicit content_type types directly to this page.. For ex: routing /ansible_collection would route to this page with contentTypeSelected reading from the URL..